service/src/main/java/bio/terra/pipelines/app/configuration/internal/PipelineConfigurations.java

```diff
@@ -0,0 +1,55 @@
+package bio.terra.pipelines.app.configuration.internal;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@Setter
+@Getter
+@ConfigurationProperties(prefix = "pipelines.configurations.pipelines")
+public class PipelineConfigurations {
```

Comment:
hmm should this now include the info in `PipelinesCommonConfiguration` ? seems like many of the places that use one also use the other.